### PR TITLE
{Packaging} Bump requests to 2.32.0

### PR DESCRIPTION
**Pull request by ThaYonghan**
**Description**
Bump requests to fix [CVE-2024-35195](https://github.com/advisories/GHSA-9wx4-h78v-vm56)
Also bump urllib3 to latest version

**History Notes**

[Core] Resolve CVE-2024-35195

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

**Comment by bebound** [comment]
@ThaYonghan Thanks for the contribution. Could you please revert the change for urllib3?

**Review by bebound — APPROVED** [review]

**Comment by ThaYonghan** [comment]
@microsoft-github-policy-service agree company="Tetra Pak"

**Review by jiasli — APPROVED** [review]

File: src/azure-cli/requirements.py3.Darwin.txt [hunk 1]
```diff
@@ -124,7 +124,7 @@ PyNaCl==1.5.0
 pyOpenSSL==24.0.0
 python-dateutil==2.8.0
 requests-oauthlib==1.2.0
-requests[socks]==2.31.0
+requests[socks]==2.32.0
```
Comment (bebound) [inline src/azure-cli/requirements.py3.Darwin.txt:127]: We use `requests[socks]` instead of `requests`, so dependabot can't create PR automatically.

Comment (ThaYonghan) [reply src/azure-cli/requirements.py3.Darwin.txt:127]: Any specific reason we don't want PRs created automatically for requests? Or should I change it to just `requests`?

**Pull request merged**